analysis_driver/pipelines/projects.py

```diff
@@ -30,33 +31,32 @@ def project_pipeline(dataset):
     if len(gvcf_files) < 2:
         raise PipelineError('Incorrect number of gVCF files: require at least two')
 
-    dataset.start_stage('relatedness')
-    working_dir = os.path.join(cfg['jobs_dir'], project_id)
     reference = cfg['references'][species]['fasta']
-    r = Relatedness(dataset, working_dir, gvcf_files, reference, project_id)
-    r.start()
-    vcftools_relatedness_expected_outfile, exit_status = r.join()
-    dataset.end_stage('relatedness')
-
-    dir_with_output_files = os.path.join(working_dir, 'relatedness_outfiles')
-    os.makedirs(dir_with_output_files, exist_ok=True)
-    files_to_symlink = output_files_config.query('project_process')
-
-    for symlink_file in files_to_symlink:
-        source = os.path.join(
-            working_dir,
-            os.path.join(*symlink_file['location']),
-            symlink_file['basename'].format(project_id=project_id)
-        )
-
-        symlink_path = os.path.join(dir_with_output_files, symlink_file['basename'].format(project_id=project_id))
-        if os.path.isfile(source):
-            if os.path.islink(symlink_path):
-                os.unlink(symlink_path)
-            os.symlink(source, symlink_path)
-        else:
-            raise PipelineError('Could not find the file ' + source + ', unable to create link')
-
-    exit_status += output_project_data(dir_with_output_files, project_id)
-
-    return exit_status
+    relatedness = Relatedness(dataset=dataset, gvcf_files=gvcf_files, reference=reference, project_id=project_id)
+    output = Output(dataset=dataset, previous_stages=[relatedness])
+    return output
+
+
+class Output(segmentation.Stage):
+    def _run(self):
+
```

Comment:
Most of this pipeline was data output, so it's not in a Stage that requires `qc.Relatedness`